packages/markdown-pdf/src/PdfTransformer.js

```diff
@@ -187,120 +215,122 @@ class PdfTransformer {
         const visitor = new ToPdfMakeVisitor(this.options);
         input.accept(visitor, parameters);
 
-        const dd = parameters.result;
-        // console.log(JSON.stringify(dd, null, 2));
+        let dd = parameters.result;
+        await findReplaceImageUrls(dd);
```

Comment:
Let's wrap this in an option (can default to true, so that it can be easily turned off.)

Reply:
I'm more worried about what happens with no network access of behind a firewall, than out of memory.

Reply:
Ok.. Will do.

Reply:
I'm thinking it might be better to allow no network access and/or unreachable URLs errors to fail gracefully.  I'm adding logic to insert a placeholder image if the `axios.get` request fails, for whatever reason - like web browser do when there's a broken image link.  We need to do this anyway.  The markdown-pdf transform will fail if we leave unresolved `![](http...)` inline.

We can add an option to turn off image URL replacement if you like but this would only really be useful if there are no image URLs in the markdown and the user is looking to save processing time when transforming a particularly large file.